src/micromamba/install.cpp

```diff
@@ -308,7 +296,6 @@ install_specs(const std::vector<std::string>& specs, bool create_env, int solver
         repo_ptrs.push_back(&r);
     }
 
-    std::cout << std::endl;
```

Comment:
?

Reply:
also done, I understand distinction made between log and output.